### PR TITLE
CI for #20067

### PR DESCRIPTION
**Pull request by eliserichards**
CI for #20067

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt [hunk 1]
```diff
@@ -336,49 +341,50 @@ open class DefaultToolbarMenu(
...
-                addToHomeScreenItem.apply { visible = ::canAddToHomescreen },
-                installToHomescreen.apply { visible = ::canInstall },
-                addToTopSitesItem,
-                saveToCollectionItem,
-                BrowserMenuDivider(),
-                settingsItem,
-                if (shouldDeleteDataOnQuit) deleteDataOnQuit else null,
-                if (shouldUseBottomToolbar) BrowserMenuDivider() else null,
-                if (shouldUseBottomToolbar) menuToolbar else null
+    var coreMenuItems = coreMenuItems()
```
Comment (MozillaNoah) [inline app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt:345]: You might be able to keep the lazy {} after this refactor

File: app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt [hunk 2]
```diff
@@ -115,6 +108,14 @@ class HomeMenu(
         val experiments = context.components.analytics.experiments
         val settings = context.components.settings
 
+        val syncSignInMenuItem = BrowserMenuImageText(
```
Comment (MozillaNoah) [inline app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt:111]: I probably missed it. Where is this being used outside of this class?

**Comment by eliserichards** [comment]
Hey @MozillaNoah this PR is just for running the CI for the contributor, can you put those comments on the PR this is linked to? 😄

**Comment by MozillaNoah** [comment]
@eliserichards, my bad. I moved them. Thanks for the callout!

**Pull request closed without merging**